src/main/java/org/jitsi/jicofo/recording/jibri/Utils.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright @ 2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.jicofo.recording.jibri;
+
+import java.util.*;
+
+/**
+ * @author bbaldino
+ */
+public class Utils
+{
+    public static String generateSessionId(int length) {
```

Comment:
in java we always put { on new line plus the docs are missing

Reply:
fixed